### ISSUE TITLE
A11y: Mobile nav menu items should be navigable via keyboard up and down keys

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#979]
## Describe the task

When the screen is narrower and the mobile nav hamburger button pops up, it is not reachable or operable via keyboard navigation. Since it's not reachable by keyboard, screenreaders are affected as well.

This ticket is the SECOND part of making the mobile nav more accessible.[ This ticket](https://github.com/CodeForPhilly/clean-and-green-philly/issues/978) must be completed first. 

1. You should be able to use UP and DOWN arrow keys to navigate the menu items.
2. Focus state should be visible
3. Pressing ENTER or SPACE will choose the option. The menu should close and you'll go to that page.

## Resources
Total expected behavior of the menu: 
[(First ticket)](https://github.com/CodeForPhilly/clean-and-green-philly/issues/978):
You should be able to TAB to the hamburger menu, press ENTER or SPACE and the dropdown opens. 

(THIS ticket)
Then you can use the UP or DOWN arrow keys to navigate the items (with focus state visible). Press ENTER or SPACE to choose one of the items. The menu should close and you'll go to that page.

[(Third ticket)
](https://github.com/CodeForPhilly/clean-and-green-philly/issues/980)While the dropdown is open pressing ESC or TAB will close the dropdown and focus will go back to the hamburger button. The page will not change.

See this example for most of the desired behavior (except for the behavior while the dropdown is open and pressing TAB.) https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/#mythical-page-content

**Comment by millmason** [comment]
Just making a note for any future implementer, since I'm working on the preceding, related ticket ( #978 ) The mobile nav menu items are currently navigable via tab as expected -- what is _not_ expected is that they are the absolute last items in tab order for the page. If you inspect the HTML of the page, they're at the very bottom of the document (explaining the tab order.) I don't know why this would be, since in the React code, the mobile menu is fully located inside of the `Header` component.

**Comment by millmason** [comment]
@bacitracin I've been looking into this a bit -- not sure what the lift is going to be like, but would you mind if I took it on?

**Comment by CodeWritingCow** [comment]
@millmason assigned! Mill and @bacitracin, #978 just went live, so this ticket should be unblocked.

**Comment by bacitracin** [comment]
@millmason @CodeWritingCow thanks for the heads up, let me know if you have any questions
